src/main/java/core/basesyntax/Triangle.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax;
+
+public class Triangle extends Shape {
+    private double leg1;
+    private double leg2;
+    private double hypotenuse;
+
+    public Triangle(double leg1, double leg2, Colour colour) {
```

Comment:
recommendation: 
it's better to have constructor without parameters as well, so it would be more comfortable to initialize object, having few options of how to create it